new/LIT6940IncarnationAmh.xml

```diff
@@ -0,0 +1,70 @@
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6940IncarnationAmh" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Commentary on the Mystery of the Incarnation in Amharic</title>
+                <title xml:lang="am" xml:id="t2">የትስብእትና፡ የመለኮት፡ ተዋህዶ፡</title>
+                <title xml:lang="am" corresp="#t2" type="normalized">Ya-tǝsbǝʾǝtǝnnā ya-malakot tawāhǝdo</title>
+                <title xml:lang="am" xml:id="t3">የምሥጢረ፡ ሥጋዌ፡ የትስብዕትና፡ የመለኮት፡ ተዋህዶ፡</title>
+                <title xml:lang="am" corresp="#t3" type="normalized">Ya-mǝśtira śǝgāwe ya-tǝsbǝʾǝtǝnnā ya-malakot tawāhǝdo</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A commentary on the mystery of Incarnation with a similar content as <ref type="work" corresp="LIT1097Ammest"/>.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="AmharicLiterature"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Theology"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-02-23">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6940IncarnationAmh" passive="BLorient2198"/>
```

Comment:
we usually use the list of witnesses (`listWit `and `witness `under `sourceDesc`) for Works attested in Manuscripts